Firmware/communication/ascii_protocol.cpp

```diff
@@ -108,15 +108,15 @@ void ASCII_protocol_process_line(const uint8_t* buffer, size_t len, StreamSink&
             if (numscan >= 3)
                 axis->controller_.input_vel_ = vel_feed_forward;
             if (numscan >= 4)
-                axis->controller_.input_current_ = current_feed_forward;
+                axis->controller_.input_torque_ = torque_feed_forward;
             axis->controller_.input_pos_updated();
             axis->watchdog_feed();
         }
 
     } else if (cmd[0] == 'q') { // position control with limits
         unsigned motor_number;
-        float pos_setpoint, vel_limit, current_lim;
-        int numscan = sscanf(cmd, "q %u %f %f %f", &motor_number, &pos_setpoint, &vel_limit, &current_lim);
+        float pos_setpoint, vel_limit, torque_lim;
+        int numscan = sscanf(cmd, "q %u %f %f %f", &motor_number, &pos_setpoint, &vel_limit, &torque_lim);
```

Comment:
Here is a good example where we do in fact want a torque limit (not current limit), as it's a proxy for an acceleration limit. I guess we should have both a torque and current limit be observed seperately.